testdata/packages/mssql-0.9.0/docs/README.md

```diff
@@ -0,0 +1,211 @@
+Note: This integration is in beta!
```

Comment:
@hbharding We will need a way to indicate that an integration is in beta / experimental.